src/api/SFA.DAS.Provider.Events.Api.Client.UnitTests/PaymentsEventsApiClient/WhenGettingPaymentStatistics.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Moq;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using SFA.DAS.Provider.Events.Api.Types;
+
+namespace SFA.DAS.Provider.Events.Api.Client.UnitTests.PaymentsEventsApiClient
+{
+    public class WhenGettingPaymentStatistics
+    {
+        private PaymentsEventsApiConfiguration _configuration;
+        private Client.PaymentsEventsApiClient _client;
+        private Mock<SecureHttpClient> _httpClient;
+
+        [SetUp]
+        public void Arrange()
+        {
+            _configuration = new PaymentsEventsApiConfiguration
+            {
+                ApiBaseUrl = "some-url/",
+                ClientToken = "super_secure_token"
+            };
+
+      
+            _httpClient = new Mock<SecureHttpClient>();
+            _httpClient.Setup(c => c.GetAsync(It.IsAny<string>()))
+                .Returns(Task.FromResult(JsonConvert.SerializeObject(new PaymentStatistics()
+                {
+                    TotalPayments = 500
+                })));
+
+            _client = new Client.PaymentsEventsApiClient(_configuration, _httpClient.Object);
+        }
+
+        [Test]
+        public async Task ThenItShouldCallTheCorrectUrl()
```

Comment:
I don't think this test adds much value.  If you replaced the It.IsAny in the Setup with a specific URL then the second test will only pass if the correct URL is used so you get this verification for free and your tests become less implementation specific.

Reply:
done